frontend/src/styles.sass

```diff
@@ -11,6 +11,8 @@ $bold-weight:     700
 $xbold-weight:    800
 $black-weight:    900
 
+$menu-z-index: 102
```

Comment:
The fact that you moved 102 to a variable doesn't mean you've fixed the issue. Let's talk on a call and I'll tell what I mean